Source/WebKit/GPUProcess/media/RemoteMediaResourceManager.h

```diff
@@ -72,7 +72,7 @@ class RemoteMediaResourceManager
     void loadFailed(RemoteMediaResourceIdentifier, const WebCore::ResourceError&);
     void loadFinished(RemoteMediaResourceIdentifier, const WebCore::NetworkLoadMetrics&);
 
-    HashMap<RemoteMediaResourceIdentifier, RemoteMediaResource*> m_remoteMediaResources;
+    HashMap<RemoteMediaResourceIdentifier, ThreadSafeWeakPtr<RemoteMediaResource>> m_remoteMediaResources;
```

Comment:
Is use of this thread safe?

Reply:
It's not but seems like this code doesn't run across threads anyway?